app/models/template_attribute.rb

```diff
@@ -14,6 +17,18 @@ def controlled_vocab?
     sample_attribute_type.base_type == Seek::Samples::BaseType::CV
   end
 
+  def allow_isa_tag_change?
+    !inherited?
+  end
+
+  def inherited?
+    !parent_attribute_id.nil?
```

Comment:
would be better to do `parent_attribute.present?` - to handle where the parent id no longer exists, and also looks cleaner